src/store/creator/types.ts

```diff
@@ -21,7 +21,8 @@ export type CreatorItemHighlight = CreatorItemBase & {
 };
 
 export type CreatorItemDrawing = CreatorItemBase & {
-    path_groups: Array<PathGroup>;
+    drawnPathGroups: Array<PathGroup>;
+    stashedPathGroups: Array<PathGroup>;
```

Comment:
What does `stashed` mean in this context? Also, it doesn't seem like this is currently used anywhere?

Reply:
Yeah it's not used yet, but basically it's the paths that have been "undo-ed" and ready to be "redo-ed"

Reply:
Sure I'll remove it for now